pkg/rpc/server/server.go

```diff
@@ -1127,7 +1127,13 @@ func (s *Server) invokescript(reqParams request.Params) (interface{}, *response.
 // runScriptInVM runs given script in a new test VM and returns the invocation
 // result.
 func (s *Server) runScriptInVM(script []byte, tx *transaction.Transaction) *result.Invoke {
-	vm := s.chain.GetTestVM(tx, nil)
+	// When transfering funds, script execution does no auto GAS claim,
+	// because it depends on persisting tx height.
+	// This is why we provide block here.
+	b := block.New(s.network, s.stateRootEnabled)
+	b.Index = s.chain.BlockHeight() + 1
+
+	vm := s.chain.GetTestVM(tx, b)
```

Comment:
You need to fix `runtimeGetTime` after this. And add some more reasonable time to the block above.

Reply:
Fixed.